docs/learn/learn-smart-contracts.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Smart Contracts on Polkadot
+description: Learn about Ethereum-compatible smart contracts n Polkadot
+---
+
+
+Smart contracts are self-executing programs that run deterministically on a blockchain, enabling decentralized applications (dApps) without intermediaries. Polkadot natively supports smart contracts written in Ethereum-compatible languages like Solidity or Vyper, as well as Rust. Additionally, the underlying Polkadot Virtual Machine allows any smart contract that compiles to RISC-V to run on its network.
+
+!!!info "Smart Contracts Documentation"
+    For detailed guides on writing and deploying smart contracts on Polkadot, refer to the official [Polkadot Developer Docs](https://docs.polkadot.com/develop/). Note that PolkaVM is under active development.
+
+## Smart Contracts vs. Rollups
+
+Polkadot offers two primary ways to build decentralized applications:
+
+- **Smart Contracts** - Deployed on existing blockchains, they are easier to develop but have limited flexibility in terms of governance and consensus.
+- **Parachains** - Independent blockchains connected to Polkadot, allowing full customization of business logic, consensus, and governance.
+
+For a deeper comparison, read:  
+[Smart Contracts vs. Parachain Development](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/reference_docs/runtime_vs_smart_contract/index.html)
+
+## Smart Contracts on Polkadot
+
+Polkadot smart contracts will reside on the **Asset Hub**. When we talk about smart contracts on Polkadot, we're specifically referring to those deployed on the Asset Hub parachain, not the relay chain itself, whose main function stays securing parachain blocks and keeping the shared state. Smart contract functionality on the Hub is made possible thanks to the **pallet-revive**, an upgraded version of the original `pallet-contracts`. Unlike its predecessor (which used Wasm), `pallet-revive` executes contracts compiled to **Polkadot Virtual Machine (PVM)** bytecode.
+
+### Key Differences:
```

Comment:
Perhaps this would be better put as a table?

Reply:
Good suggestion! We can propose this later on.